lib/stream_data.ex

```diff
@@ -379,14 +391,11 @@ defmodule StreamData do
     end)
   end
 
+  # QUESTION: When would I use the function below?
```

Comment:
It's used for things such as `byte/0` where `byte/0` depends on the implementation of `int/1` which shrinks, but it doesn't make sense for bytes to shrink. Probably, we can make this function private for now but it's worth noting that it's not possible to do what it does with the rest of the public API. Thoughts @josevalim?